File: week1/pset1/cash.c
//Variables = (total <- dinero recibido
//                cents <- dinero rec en cents
//              remain <- cents inic - q or d or n or p
//            q <- cantidad de quarters
//          d <- c. de dimes
//        n <- c. de nickels
//      p <- c. de pennies

#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float total;
    int cents;
    int remain;
    int q = 0;
    int d = 0;
    int n = 0;
    int p = 0;
    do
    {
        total = get_float("Change owed: ");
    }
    while(total <= 0);
    cents = round(total*100);
    remain = cents;
    
    while (remain >= 25)
    {
        q++;
        remain -= 25;
    }
    while (remain >= 10)
    {
        q++;
        remain -= 10;
    }
    while (remain >= 5)
    {
        q++;
        remain -= 5;
    }
    while (remain >= 1)
    {
        q++;
        remain -= 1;
    }
    printf("%d\n", q);
}

File: week6/pset6/cash/cash.py
from cs50 import get_float

while True:
    amount = get_float("Change owed: ")
    if amount > 0:
        break
    
plata = round(amount * 100)
monedas = 0
remainder = plata

while(plata > 0):
    if plata >= 25:
        plata -= 25
        monedas += 1
    elif plata >= 10:
        plata -= 10
        monedas += 1
    elif plata >= 5:
        plata -= 5
        monedas += 1
    else:
        plata -= 1
        monedas += 1
print(monedas)